src/pages/ProductDetail/BeforeCheck/BeforeCheck.js

```diff
@@ -0,0 +1,140 @@
+import React, { useState } from 'react';
+import styled from 'styled-components';
+import { FiChevronDown } from 'react-icons/fi';
+import { FiChevronUp } from 'react-icons/fi';
+
+function BeforeCheck() {
+  const [toggle, setToggle] = useState([false, false, false]);
```

Comment:
```suggestion
  const [currentNotice, setCurrentNotice] = useState(0);
```

currentNotice는 현재 열려 있는 공지의 아이디 값을 다루는 state입니다.
현재 CONFIRM의 데이터를 보니 아이디가 1,2,3으로 지정이 되어있네요!

해당 state와 선택한 공지의 아이디가 같은 것만 열리도록 구현하고
다시 누르면 해당 state가 0이 되도록 구현하면 복잡한 로직 없이 구현할 수 있겠습니다!